### DISCUSSION TITLE
Can we have option to get the build list manually

### DISCUSSION THREAD

**Discussion by bn3d** [Victor-IX/Blender-Launcher-V2#69] [Ideas]
Hello,
Thank you for picking up the torch for this extremely useful software!

I was wondering since there was a problem with the scraping of Blender's official download page supposedly generating too much traffic (it seems to be fixed now, but still..) - can we get an option in the Settings that disables all automatic checks and forces a scrape manually. Now we have two ways of getting the builds - automatic in some predefined interval (that is disabled by default) and on startup. Since most of the times I run Blender-Launcher just to start Blender or particular build it is unnecessary to update the build list on startup or automatically and generate traffic towards the official Blender website.

The way I think this could be implemented is to change "Check For New Builds Automatically" into a drop-down menu "Check For New Builds" with three options:

1. On startup - current default
2. Automatic - when chosen you get the text box that defines the time interval
3. Manual - the user initiates an build list update by clicking the update button in the bottom left of the main window

**Comment by zeptofine** [comment]
In the latest version, the automatic build check has been forcibly disabled inside but the option to turn it on is still there (it does nothing 😬)

I like the dropdown idea but it feels like in the future there is a world where people might want the on startup option and the automatic option at the same time. I've added a toggle in the main branch currently, but i will look for prettier ways to convey this information

**Comment by bn3d** [comment]
Thank you for implementing the 'Check on startup' option in version 2.1.24!
